frameworks/Python/bottle/uwsgi.ini

```diff
@@ -2,7 +2,7 @@
 master
 ; Increase listen queue used for nginx connecting to uWSGI. This matches
 ; net.ipv4.tcp_max_syn_backlog and net.core.somaxconn.
-listen = 65535
+listen = 128
```

Comment:
128 is enough for concurrency=128, off course.
And it may be not enough for concurrency=256, but I don't know until measure.

Because it uses unix domain socket, backlog shortage cause "502 Bad Gateway" soon.
I'll think about it after seeing next result.

@knewmanTE Could you ensure `net.core.somaxconn` is more than 1024 for all setup?